pkg/database/database.go

```diff
@@ -96,16 +95,13 @@ func NewDb(appConfig *config.AppConfig, m monitoring.MetricsI) Db {
 	return dbInst
 }
 
-// Status checks if the database is reachable with a 5 sec timeout
+// Status a single cluster
 func (d *db) Status() error {
 	params := &dynamodb.DescribeTableInput{
 		TableName: &d.table.name,
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Is there a reason for removing the timeout on the Database status?

Reply:
hmm, nope. Probably by accident 😄